packages/sync-actions/src/discount-codes.js

```diff
@@ -13,6 +14,11 @@ function createDiscountCodesMapActions(mapActionGroup) {
     allActions.push(
       mapActionGroup('base', () => actionsMapBase(diff, oldObj, newObj))
     )
+
+    allActions.push(
+      mapActionGroup('custom', () => actionsMapCustom(diff, newObj, oldObj))
```

Comment:
Now the two actions map functions have different signatures 😔 probably time for a bigger refactor?

Reply:
Yeah but that mix and confusion with `old` and `new` is beyond me. It's flipped here and there and different in the docs.